### ISSUE TITLE
CIBA spec: Missing document reference

### ISSUE THREAD

**Issue by marcusalmgren** [OpenBanking-Brasil/specs-seguranca#406]
The CIBA spec makes a set of references to [FAPI-BR], but this document is not mentioned among the references.

**Comment by marcusalmgren** [comment]
Another tiny thing is that I believe the document is called "open-banking-brasil-financial-api-CIBA-1_**ID1**.md" but it is actually ID3.

**Comment by guilhermedecampo** [comment]
Yes, the link for the FAPI BR was added also in the list of CIBA documentation on the developers area.
